### PR TITLE
Tokens - show manage flag

### LINKED ISSUE #1783: Show info if token can/cannot manage other tokens
Related to: https://github.com/keboola/connection/pull/1379#issuecomment-396933018

### PR DESCRIPTION
**Pull request by ujovlado**
Fixes #1783 

Proposed changes:

- add section about "canManageTokens" to tokens list and token detail

screenshot_2018-06-18_15-28-18
screenshot_2018-06-18_15-28-38

### CODE REVIEW

**Comment by ujovlado** [comment]
@Halama dal som req. na teba, riesili sme to myslim spolu na slacku

**Comment by ujovlado** [comment]
Este to chce jednu upravu

**Comment by ujovlado** [comment]
screenshot_2018-06-19_11-58-51

File: src/scripts/modules/tokens/react/Index/TokensTable.jsx [hunk 1]
```diff
@@ -268,7 +268,11 @@ export default React.createClass({
           {this.renderBucketsAccess(token)}
         </div>
         <div className="td">
-          {token.get('canManageBuckets', false) ? 'Yes' : 'No'}
+          {token.get('canManageBuckets', false) ? (
+            <i className="fa fa-check" />
```
Comment (Halama) [inline src/scripts/modules/tokens/react/Index/TokensTable.jsx]: https://github.com/keboola/indigo-ui/blob/master/src/indigo/components/Check.jsx ? :)

Comment (ujovlado) [reply src/scripts/modules/tokens/react/Index/TokensTable.jsx]: :-D OMG

**Comment by ujovlado** [comment]
Tak s tymto PR som teda vol. Sorry.

**Comment by Halama** [comment]
jsem si nakonec rekl ze kouknu i do kodu a zkusim to :)

**Comment by ujovlado** [comment]
Len pre info, fixol som to potom v mastri ...

**Pull request merged**